src/main/groovy/org/testatoo/core/internal/Identifiers.groovy

```diff
@@ -58,7 +62,7 @@ class Identifiers {
 
         if(!cachedComponents.get(clazz)) {
             List<Class> matchingClasses = new ArrayList<>()
-            scanner.matchSubclassesOf(clazz, { c -> matchingClasses.add(c) }).scan()
+            matchingClasses.addAll(scan.getSubclasses(clazz.name).loadClasses())
```

Comment:
Not a big deal, but you could simplify this to just:

```java
cachedComponents.put(clazz, scan.getSubclasses(clazz.name).loadClasses());
```

i.e. you can just store the `List<Class>` that `loadClasses()` returns, directly -- right now you're creating another list and initializing it as a copy of the returned list.

Reply:
Thank you, fixed.